.github/CODEOWNERS

```diff
@@ -77,6 +77,11 @@ net/proxy_resolution/ @brave/tor-reviewers
 chromium_src/**/extensions/**/*_features.json @brave/sec-team
 chromium_src/tools/json_schema_compiler/ @brave/sec-team
 
+# Browser commands available via Brave Education content
+browser/ui/webui/brave_browser_command/brave_browser_command_handler.* @brave/sec-team
```

Comment:
sec-team should also have codeowner for the ts/js code that handles the commands

Reply:
also I think only the `.cc` matters here for sec-team

Reply:
fixed